templates/corporate/support_request_thanks.html

```diff
@@ -1,13 +1,19 @@
 {% extends "zerver/portico.html" %}
 
+{% block title %}
+<title>{{ _("Zulip: Thanks for contacting us") }}</title>
+{% endblock %}
+
 {% block portico_content %}
 <div class="flex full-page thanks-page">
     <div class="center-block new-style">
-        <h1>Thanks for contacting us!</h1>
-        <p>We will be in touch with you soon.</p>
+        <h1>{{ _("Thanks for contacting us!") }}</h1>
+        <p>{{ _("We will be in touch with you soon.") }}</p>
         <p>
+            {% trans %}
             You can find answers to frequently asked questions in the
             <a href="/help/">Zulip help center</a>.
+            {% endtrans %}
```

Comment:
Noted that this template wasn't translated, though that may have been intentional since most support replies will be in English?